packages/studio/src/components/VersionNumber.tsx

```diff
@@ -0,0 +1,58 @@
+import { useCallback } from "react";
+import { version } from "../../package.json";
+import { ReactComponent as Info } from "../icons/info.svg";
+import ButtonWithModal, { renderModalFunction } from "./common/ButtonWithModal";
+import ReactModal from "react-modal";
+
+const customReactModalStyles = {
+  content: {
+    top: "50%",
+    left: "50%",
+    right: "50%",
+    bottom: "auto",
+    marginRight: "-95%",
+    maxWidth: "600px",
+    transform: "translate(-50%, -50%)",
+  },
+  overlay: {
+    zIndex: 1000,
```

Comment:
do we specify z-index in the other modals?

Reply:
I think we do in all our custom modals

Reply:
oh nevermind I see this is copied from Modal.tsx, could we turn the current Modal.tsx into DialogModal.tsx (or something), and have it use a more general, basic modal component that we also use here??

Reply:
oop didn't see your message in the middel

Reply:
done!